content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -0,0 +1,431 @@
+===========
+Help center
+===========
+
+Odoo *Helpdesk* integrates with the *Forums*, *eLearning*, and *Knowledge* apps to create the *help
+center*.
+
+.. image:: help_center/help-center-enable-features.png
+   :align: center
+   :alt: Overview of the settings page of a team emphasizing the help center features.
+
+The *help center* is a centralized location where teams and customers can search for (and share)
+detailed information about products and services.
+
+.. note::
+   In order to activate any of these features on a *Helpdesk* team, (*Forums*, *eLearning*, or
+   *Knowledge*), the :guilabel:`Visibility` of the team has to be set to :guilabel:`Invited portal
+   users and all internal users`. See :doc:`Getting Started
+   </applications/services/helpdesk/overview/getting_started>` for more information on *Helpdesk*
+   team settings and configuration.
+
+.. warning::
+   Since all of the *help center* features require integration with other applications, enabling any
+   of them may result in the installation of additional modules (or applications).
+
+   Installing a new application on a *One-App-Free* database will trigger a 15-day trial. At the end
+   of the trial, if a paid subscription has not been added to the database, it will no longer be
+   active or accessible.
+
+Knowledge
+=========
+
+Odoo's *Knowledge* application is a collaborative library where users can store, edit, and share
+information. The *Knowledge* app is represented throughout the database by a *book* icon.
+
+.. image:: help_center/help-center-knowledge-book-icon.png
+   :align: center
+   :alt: View of a message in Helpdesk focusing on the Knowledge book icon.
+
+Enable knowledge on a Helpdesk team
```

Comment:
wondering if we do uppercase here, since what we're saying here is "Integrate the Knowledge App with a team from the Helpdesk app"

wdyt?

```suggestion
Enable Knowledge on a Helpdesk team
```